appcli/app_test.go

```diff
@@ -263,7 +264,7 @@ func TestGetConfigMetadata_quiet(t *testing.T) {
 		)
 	}
 
-	if !metadata.Quiet {
+	if metadata.Verbosity != ui.VerbosityLevelQuiet {
 		t.Errorf(
 			"GetConfigMetadata(%s): expected Quiet: true, actual: false",
```

Comment:
Make sure the message here reflects the actual assertion.

You can use the example code snippet from my comment above to get an idea of the format.